rllib/models/tf/tf_distributions.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, *args, **kwargs):
         self._dist = self._get_tf_distribution(*args, **kwargs)
 
     @abc.abstractmethod
-    def _get_tf_distribution(self, *args, **kwargs) -> tfp.distributions.Distribution:
+    def _get_tf_distribution(self, *args, **kwargs) -> "tfp.distributions.Distribution":
```

Comment:
did we need to introduce this in this test?

Reply:
Yes, since we imported Catalog the dependency_tf / torch stuff are failing. I have done some other stuff on top of it to hopefully get around this problem in the test. Let's see if the tests pass now.